moveit_core/constraint_samplers/src/constraint_sampler.cpp

```diff
@@ -36,13 +36,15 @@
 
 #include <moveit/constraint_samplers/constraint_sampler.h>
 
+rclcpp::Logger LOGGER_CONSTRAINT_SAMPLERS = rclcpp::get_logger("moveit").get_child("constraint_samplers");
```

Comment:
Refactor to put this inside namespace. the methods themselves could be put in a single large namespace block as well

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/1611306fdc5f8fcf2b415b3aab202a25db9ee351